views/cc.hbs

```diff
@@ -1,13 +1,15 @@
 <html>
 <head>
+    <title>OIDC Doctor - Client-Credential-Flow</title>
+    <link rel="icon" href="/images/logo-amos.ico" type="image/x-icon">
     <meta charset="utf-8" />
-    <title>Client-Credential-Flow</title>
 
     <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.1.3/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-1BmE4kWBq78iYhFldvKuhfTAU6auU8tT94WrHftjDbrCEXSU1oBoqyl2QvZ6jIW3" crossorigin="anonymous">
 </head>
 <body>
     <nav class="navbar">
         <div class="container">
+        <img class="mx-auto d-block" src="/images/logo-amos.png" alt="Image"  width="430" height="300">
```

Comment:
The icon in the navbar is too large and displayed at the wrong position: 
![grafik](https://user-images.githubusercontent.com/25197252/179422292-08312f21-e3b5-4e26-bbfe-0355e4b050ef.png)

Reply:
Then I will reduce the size more.